code/game/objects/items/storage/bags.dm

```diff
@@ -162,11 +162,17 @@
 		playsound(user, SFX_RUSTLE, 50, TRUE)
 
 		if (box)
-			user.visible_message(span_notice("[user] offloads the ores beneath [user.p_them()] into [box]."), \
-				span_notice("You offload the ores beneath you into your [box]."))
+			balloon_alert(user, "scoops ore into box")
```

Comment:
this message (and the other) feels slightly off to me but i'm not sure what would be better

Reply:
Since Balloon alerts need to be concise most of them sound weird.

These two "work" but I was hoping someone could come up with a clearer ones to be honest.